Remove unnecessary comments on issue templates (#4557)
ppy/osu: .github/ISSUE_TEMPLATE/missing-for-live-issues.md

name: Missing for Live

> about: Let us know the features you need which are available in osu-stable but not lazer

Reviewer: ```suggestion
about: Features which are available in osu!stable but not yet in osu!lazer.
```

---
**Describe the missing feature:** 

> **Designs:** 

Reviewer: ```suggestion
**Proposal designs of the feature:** 
```
